src/patternfly/components/Tile/tile.scss

```diff
@@ -142,13 +132,13 @@
     justify-content: initial;
 
     .pf-c-tile__icon {
-      display: flex;
-      justify-content: center;
-      margin-bottom: var(--pf-c-tile__header--m-stacked__icon--MarginBottom);
-
       --pf-c-tile__icon--MarginRight: 0;
       --pf-c-tile__icon--FontSize: var(--pf-c-tile__header--m-stacked__icon--FontSize);
       --pf-c-tile__icon--Color: var(--pf-c-tile__header--m-stacked__icon--Color);
+
+      display: flex;
+      justify-content: center;
```

Comment:
you probably want to add `align-items: center;` here, or at least on the display-lg.stacked icon selector since that has a defined height and the image may be shorter than the height